### PR TITLE
Twilson63/hyper test integration 434

### PR DESCRIPTION
**Pull request by twilson63**
Got Storage and Queue - hyper-tests implemented and working

- [x] - works for nano version of hyper
- [x] - works for cloud version of hyper

### CODE REVIEW

**Comment by twilson63** [comment]
HOLD on looking into CI failures

**Comment by twilson63** [comment]
Ready for review, found one issue that needs to be resolved on hyper connect in a separate issue.

`remove` for `storage` service needs to be implemented.

File: packages/test/lib/utils.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ export const toJSON = (res) => {
   return res.ok
     ? Async.fromPromise(res.json.bind(res))()
     : Async.fromPromise(res.json.bind(res))()
+      .map((r) => (console.log(r), r))
```
Comment (TillaTheHun0) [inline packages/test/lib/utils.js:11]: Do we want to keep these logs?

File: packages/test/mod.js [hunk 2]
```diff
@@ -20,6 +18,9 @@ if (!ci) {
   ]);
 }
 const hyperCS = answers.hyper === "" ? cs : answers.hyper;
+const isCloud = /^cloud/.test(answers.hyper);
+console.log("isCloud", isCloud);
```
Comment (TillaTheHun0) [inline packages/test/mod.js:22]: keep?

File: packages/test/search/bulk.js [hunk 3]
```diff
@@ -20,6 +20,7 @@ export default function (search) {
   test("POST /search/:index/_bulk", () =>
     $fetch(() => search.load(docs))
       .map((r) => {
+        console.log("bulk", r);
```
Comment (TillaTheHun0) [inline packages/test/search/bulk.js:23]: keep?

**Review by TillaTheHun0 — APPROVED** [review]
Looks great. I just had questions about the extra `console.log`s added. If we want to keep those, then LGTM. Otherwise, remove, push, and should be good to go.

**Pull request merged**